### PR TITLE
feat - add avatar component page to docs

### PR DESCRIPTION
**Pull request by mcmonojit**
**Added avatar component page to documentation**
- Image avatar
- Letter avatar
- Icon avatar

### CODE REVIEW

File: docs/avatar/avatar.html [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+                          <p class="text-center">small</p>
+                        </div>
+                </div>
+
+                <p>The following code snippet contains the code for the above components.</p>
+
+                <div class="component-container code-snip-width">
+                    <iframe
+                        src="https://carbon.now.sh/embed/bx3soADnFgEXJJEKOdIJ"
+                        style="width: 46rem; height: 24rem; border:0; transform: scale(1); overflow:hidden;"
```
Comment (pratik-kumar-sahu) [inline docs/avatar/avatar.html:99]: - Avoid inline styling if possible

File: docs/docs.css [hunk 2]
```diff
@@ -116,6 +120,23 @@ h1,h3{
...
+    justify-content: space-evenly;
+}
+
+.margin-r-2rem{
+    margin-right: 2rem;
+}
+
 .footer{
     padding: 1rem 2rem;
     color: beige;
```
Comment (pratik-kumar-sahu) [inline docs/docs.css:142]: use colors as variables

**Review by pratik-kumar-sahu — APPROVED** [review]
Else looks good, please go ahead with merging

**Review by NeerajVodala — APPROVED** [review]
Comments same as Pratik's. Rest everything looks good to me.
All the best.

**Pull request merged**